modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -24,7 +24,11 @@ ifeval::["{context}" == "about-ovn-kubernetes"]
 
 |IPsec encryption for intra-cluster communication|Supported|Not supported
 
-|IPv6|Supported ^[3]^ ^[4]^|Not supported
+|IPv4-primary dual-stack|Supported|Supported ^[3]^
+
+|IPv6 single-stack|Supported ^[4]^|Unsupported
```

Comment:
- v6-primary supported on baremetal was from 4.13? 
- v6-primary supported on VMWARE until 4.14?
- Any dual-stack support on OpenStack was added around 4.14?
 Dual stack of any sort on OpenStack was added around 4.14 as

Reply:
openstack only supports dualstack either ipv4 or ipv6 primary with ovn-kubernetes.